twitter.css

```diff
@@ -0,0 +1,362 @@
+    align-items: center;
+}
+
+.profile {
+    display: grid;
+    grid-template-columns: 70px 1fr;
+    column-gap: 20px;
+    align-items: center;
+}
+
+.pfp {
+    width: 70px;
+    height: 70px;
+    border-radius: 50%;
+    justify-content: center;
+}
+
+.profile-info {
+    display: grid;
+    grid-template-rows: max-content max-content;
+}
+
+.profile-info h4 {
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.profile-info p {
+    font-size: 15px;
+    color: gray;
+}
+
+
+.main-body {
+    grid-area: 1 / 2 / 3 / 3;
+    display: grid;
+    grid-template-rows: 300px 1fr;
+    row-gap: 30px;
+    border-left: solid 1px gray;
+    border-right: solid 1px gray;
+}
+
+.main-body-header {
+    position: sticky;
+    top: 0;
+    display: grid;
+    grid-template-rows: repeat(3, 100px);
+    align-items: center;
+    border-bottom: solid 1px gray;
+    background-color: white;
+    filter: drop-shadow(0px 5px 5px rgb(182, 182, 182));
+}
+
+.title {
+    font-size: 30px;
+    font-weight: bold;
+    border-bottom: solid 1px gray;
+    padding-left: 20px;
+}
+
+.tweet-box {
+    display: grid;
+    grid-template-columns: 70px 1fr;
+    column-gap: 20px;
+    align-items: center;
+    font-size: 30px;
+    color: gray;
+    padding: 0 20px;
+}
+
+.upload-icons {
+    display: inline-flex;
+    align-items: center;
+    padding-left: 100px;
+    padding-right: 20px;
+}
+
+.upload-icons .logo {
+    width: 30px;
+    height: 30px;
+    margin: 10px;
+}
+
+.upload-icons .logo:hover {
+    padding: 5px;
+    border-radius: 50%;
+    background-color: rgb(207, 207, 207);
+    transition: 0.5s;
+}
+
+.tweet {
+    margin-left: 400px;
+}
+
+.tweet button {
+    width: 130px;
+    height: 50px;
+    font-size: 15px;
+    padding: 10px;
+}
+
+.tweet button:hover {
+    transition: 0.5s;
+    background-color: rgb(0, 97, 136);
+}
+
+.main-feed {
+    display: grid;
+    grid-template-rows: repeat(3, max-content);
+    grid-auto-rows: minmax(100px, auto);
+    background-color: white;
+}
+
+.post {
+    height: max-content;
+    padding: 20px;
+    border-bottom: solid 1px gray;
+}
+
+.tweet-icons {
+    display: inline-flex;
+    gap: 100px;
+    align-items: center;
+    justify-content: space-evenly;
+    padding-top: 10px;
+    padding-left: 100px;
+    padding-right: 20px;
+}
+
+.tweet-icons .logo {
+    width: 30px;
+    height: 30px;
+    margin: 10px;
+}
+
+.feed {
+    display: grid;
+    grid-template-columns: 70px 1fr;
+    column-gap: 20px;
+    align-items: center;
+    font-size: 25px;
+    color: gray;
+    padding: 0 20px;
+}
+
+.feed .pfp {
+    grid-area: 1 / 1 / 3 / 2;
+}
+
+.feed > .lower {
+    grid-area: 2 / 2 / 3 / 3;
+    font-size: 20px;
+    color: black;
+}
+
+.right-nav {
+    display: grid;
+    grid-template-rows: 100px 500px 500px;
+    grid-template-columns: 400px;
+    gap: 20px;
+    position: sticky;
+    top: 0;
+    padding: 20px;
+    background-color: white;
+}
+
+@media screen and (max-width: 1900px) {
```

Comment:
Try to use all of the @media in the end of the css file it will be helpfull for you to reduce a lot of errors because sometimes @media can stop working if you are using them on the middle of the css file and mixing them with normal styles :)